src/sentry/demo/data_population.py

```diff
@@ -1491,3 +1653,4 @@ def handle_mobile_scenario(
             )
         self.assign_issues()
         self.inbox_issues()
+        self.generate_alerts(android_project, 17, 23)
```

Comment:
@roggenkemper so hardcoding the thresholds isn't going to work well if we ever change the scaling factor. What we need instead is a way to determine the ideal thresholds based on the actual errors we send. That's why I suggested just doing the critical alerts: we can set the threshold to be the max value of errors we get in the alert window.